packages/fullstack/fullstack.txt

```diff
@@ -3,79 +3,118 @@ autofill
 autogenerated
 autoprefixer
 baseref
+behaviour
+charlist
+checkboxes
 classlist
 codelyzer
 collapsable
 compat
-cond
```

Comment:
Is there reason why `cond` was removed?
Was it a misspelling of `conda` or is there another reason?

Reply:
Not at all, I'll fix that. Thx for the feedback